### PR TITLE
Add user ID in message content for mod-alerts, but not for autobans

### LINKED ISSUE #2088: Show user ID outside embed for `#mod-alerts`
Many moderators moderate from iOS and Android mobile devices. As of today, Discord iOS doesn't allow for copying embed text *at all*, and Android iOS allows copying of embed text as a whole (not selectively).

One of the biggest pain points for moderators is not being able to grab the user's ID from the mod-alert embed, neither from the embed itself nor from the mention of the user (because the mention doesn't resolve properly).

A quick easy fix for this is to simply add the user ID outside the embed as plain text so that moderators can copy/paste that text and research and infract the user easily. There are other proposed ideas such as buttons and such, but that can all be planned later, whereas this is a very quick easy change with a lot of immediate benefit.

I would like the user ID to be added for all mod-alerts, even non-pinging ones (such as for those sent to bot via DM)

image

### PR DESCRIPTION
**Pull request by minalike**
Continued work on #2096 
Further closes #2088 

This PR 
- additionally adds the user ID for antispam alerts
- additionally adds the user ID for username filtering alerts
- removes the user ID for autoban alerts

I'll make an additional commit to rename the `content` in the code block below, since it overlaps with the `content` parameter used a little later on.

I didn't quite understand what lines 98-101 were doing though, and need some guidance on good rename suggestions for `content` 😄 
https://github.com/python-discord/bot/blob/389cb815a379feca1122df12b3492144b76e7749/bot/exts/filters/antispam.py#L93-L101

### CODE REVIEW

**Comment by minalike** [comment]
Will skip the additional commit and my curiosity about the above code is satisfied. Please review!

File: bot/exts/filters/antispam.py [hunk 1]
```diff
@@ -103,6 +103,7 @@ async def upload_messages(self, actor_id: int, modlog: ModLog) -> None:
             mod_alert_message += content
 
         await modlog.send_log_message(
+            content=", ".join(str(m.id) for m in self.members),  # quality-of-life improvement for mobile moderators
```
Comment (ChrisLovering) [inline bot/exts/filters/antispam.py:106]: Make this a set, so the same user_id isn't sent more than once.

```suggestion
            content=", ".join({str(m.id) for m in self.members}),  # quality-of-life improvement for mobile moderators
```

File: bot/exts/filters/filtering.py [hunk 2]
```diff
@@ -423,9 +424,12 @@ async def _send_log(
             # Allow specific filters to override ping_everyone
             ping_everyone = Filter.ping_everyone and _filter.get("ping_everyone", True)
 
-        # If we are going to autoban, we don't want to ping
+        content = str(msg.author.id)  # quality-of-life improvement for mobile moderators
+
+        # If we are going to autoban, we don't want to ping and don't need the user ID
         if reason and "[autoban]" in reason:
             ping_everyone = False
+            content = ""
```
Comment (ChrisLovering) [inline bot/exts/filters/filtering.py:432]: To be explicit.
```suggestion
            content = None
```

**Review by ChrisLovering — CHANGES_REQUESTED** [review]

File: bot/exts/filters/antispam.py (hunk 1 above)
Comment (ChrisLovering) [reply bot/exts/filters/antispam.py:106]: Actually, members is already a frozenset, so this can be ignored.

**Review by Akarys42 — APPROVED** [review]

File: bot/exts/filters/filtering.py (hunk 2 above)
Comment (minalike) [reply bot/exts/filters/filtering.py]: https://github.com/python-discord/bot/pull/2098/commits/6c561e251be3a4af2af349ba4b86b9ff0f0a9b7b

**Review by ChrisLovering — APPROVED** [review]

**Review by vivekashok1221 — APPROVED** [review]
lgtm.

**Pull request merged**